doc/sphinx/addendum/micromega.rst

```diff
@@ -250,6 +250,8 @@ obtain :math:`-1`. By Theorem :ref:`Psatz <psatz_thm>`, the goal is valid.
 
 .. [#] Support for :g:`nat` and :g:`N` is obtained by pre-processing the goal with
   the ``zify`` tactic.
+.. [#] Support for :g:`Z.div` and :g:`Z.modulo` may be obtained by pre-processing the goal with
+  the ``Z.div_mod_to_quot_rem`` tactic after manually running ``zify``.
```

Comment:
This would be clearer without the passive voice.  Something like:

```.. [#] To use :g:`Z.div` and :g:`Z.modulo`, preprocess the goal with  the ``Z.div_mod_to_quot_rem`` tactic after using ``zify``.```

I'm guessing "manually" is not really needed.

Reply:
I used passive voice to mimic the bullet point above.  Saying `manually` is needed, because, in contrast with automatic support for `N` and `nat`, the user must run `zify` explicitly if they want to support div and mod mixed with `N` and `nat`.  However, this is not required in all cases (in particular, if the equations all only mention `Z`).  I don't think your phrasing is fully accurate; you can *use* `Z.div` and `Z.modulo` without the tactics; the only issue is that `nia` and `lia` won't be able to do anything with them.

I have adjusted the end of the sentence to say "(you may need to manually run `zify` first)".

Reply:
OK